String-xyz/unity-sdk
# Enable Toggle Saved Payments

To test:

1) Import this branch locally in Unity Package Manager in the Unity Demo
2) Run the Demo, complete all of the steps and execute a transaction
3) Add a call to StringXYZ.GetCards() in the Login function in GameLogicBehavior.cs (after stringPlayerID = response.user.id on line ~231)
```
        var savedCards = await StringXYZ.GetCards();
        foreach (CustomerInstrument card in savedCards)
        {
            Debug.Log($"Saved Card: {card}");
        }
```
4) Restart the Demo and log in again
5) Note that the GetCards endpoint is returning data (though at this point due to a change in Checkout it is not complete)
6) Add the "false" parameter to the StringXYZ.Transact call in ExecuteLastQuote()
7) Transact again
8) The number of results from GetCards should remain the same
